src/main/resources/db/changelog/2022.09/08-01-changelog.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-latest.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet author="rlong (generated)" id="1662577972070-1">
+        <addDefaultValue
+            columnName="soft_deleted"
+             defaultValueBoolean="false"
+             tableName="terr_assignment"
+             columnDataType="BIT(1)"/>
```

Comment:
The columnDataType should be BOOLEAN, just to be more database agnostic.

Reply:
Done!